src/main/java/pl/edu/zut/mad/schedule/search/ScheduleSpecification.java

```diff
@@ -21,10 +21,12 @@
     @Override
     public Predicate toPredicate(Root<Schedule> root, CriteriaQuery<?> query, CriteriaBuilder builder) {
 
-        if (searchCriteria.getKey().equals(Schedule.Fields.DATE_FROM.getKey())) {
+        if (searchCriteria.getKey().equals(DATE_FROM.getKey())) {
```

Comment:
I think that `searchCriteria.getKey()` could be extracted to variable, at this moment there are three places where it's called.